### PR TITLE
Thunder 6 update instructions

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

**Review by dbosen — CHANGES_REQUESTED** [review]
We should add some instructions, how to test your custom codebase, if it is Drupl 9 compatible

File: _includes/update36/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+This should update to Thunder 8.3.5 or greater.
+
+Now run database updates:
+```
+cd docroot
+drush updb
+```
+You should at least see the thunder 8308 schema update. If not, double check that the correct version of Thunder is installed, and that `drush updb` did not throw any errors.
+
+Before you start with the code and database update please disable the libraries module. The libraries module is not Drupal 9 compatible and
```
Comment (dbosen) [inline _includes/update36/prerequisites.md:26]: We should move this to the beginning, as the first step before composer update

File: _includes/update36/composer-adjustments.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+We removed some modules from our codebase. In case you are using one of
+below mentioned modules please require them manually for your project.
```
Comment (dbosen) [inline _includes/update36/composer-adjustments.md:2]: Where is this list?

**Review by dbosen — CHANGES_REQUESTED** [review]

File: _includes/update36/composer-adjustments.md (hunk 2 above)
Comment (chrfritsch) [reply _includes/update36/composer-adjustments.md:2]: It's the entity module

**Review by dbosen — DISMISSED** [review]

**Comment by chrfritsch** [comment]
We should add https://github.com/caciobanu/behat-deprecation-extension for behat tests

**Review by dbosen — APPROVED** [review]

**Pull request merged**